src/test/kotlin/org/dataloader/KotlinExamples.kt

```diff
@@ -0,0 +1,39 @@
+package org.dataloader
+
+import org.junit.jupiter.api.Test
+import java.util.concurrent.CompletableFuture
+
+/**
+ * Some Kotlin code to prove that are JSpecify annotations work here
+ * as expected in Kotlin land.  We don't intend to ue Kotlin in our tests
+ * or to deliver Kotlin code in the java
+ */
+class KotlinExamples {
```

Comment:
Just enough Kotlin - tests only

confirmed the build .jar has no Kotlin classes!